gulpfile.js

```diff
@@ -18,7 +19,8 @@ var BUILD_TASKS = [
   'copy-dirs',
   'less',
   'webpack',
-  'generate-index-files'
+  'generate-index-files',
+  'server'
```

Comment:
oh, I don't think this is something we want in `BUILD_TASKS`... that array is only meant to hold all the tasks necessary to build the static site... I should probably add a comment or rename the constant to make that more obvious, sorry!